### PR TITLE
Support "certbot-auto --no-bootstrap"

### PR DESCRIPTION
**Pull request by pde**
Support a `--no-bootstrap` flag to `certbot-auto`. This is particularly useful when running unstable OS releases where re-installing `python`, `virtualenv`, `gcc` etc is definitely not a no-op.

### CODE REVIEW

File: letsencrypt-auto-source/letsencrypt-auto.template [hunk 1]
```diff
@@ -53,6 +53,8 @@ for arg in "$@" ; do
       # Do not upgrade this script (also prevents client upgrades, because each
```
Comment (bmw) [inline letsencrypt-auto-source/letsencrypt-auto.template]: You should mention `--no-bootstrap` in the help text above.

**Comment by bmw** [comment]
I ran test farms on this and everything passed. @pde, if you're happy with this, feel free to merge (not squash).

**Comment by bmw** [comment]
@pde, want to take a look at this?

**Comment by pde** [comment]
Wow this really got forgotten. LGTM, merging.

**Pull request merged**